net48/RazorpayClient48.csproj

```diff
@@ -0,0 +1,199 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
why we not adding DefaultTargets key?

Reply:
added 

https://github.com/razorpay/razorpay-dot-net/blob/10148cd90cfb84ff83d6f4e8643488f5fe22eef5/net48/RazorpayClient48.csproj#L2